app/src/sharedTest/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdaptersTest.kt

```diff
@@ -0,0 +1,202 @@
+import org.oppia.android.domain.oppialogger.LoggingIdentifierModule
+import org.oppia.android.domain.oppialogger.analytics.ApplicationLifecycleModule
+import org.oppia.android.domain.oppialogger.analytics.CpuPerformanceSnapshotterModule
+import org.oppia.android.domain.oppialogger.logscheduler.MetricLogSchedulerModule
+import org.oppia.android.domain.oppialogger.loguploader.LogReportWorkerModule
+import org.oppia.android.domain.platformparameter.PlatformParameterModule
+import org.oppia.android.domain.platformparameter.PlatformParameterSingletonModule
+import org.oppia.android.domain.question.QuestionModule
+import org.oppia.android.domain.topic.PrimeTopicAssetsControllerModule
+import org.oppia.android.domain.workmanager.WorkManagerConfigurationModule
+import org.oppia.android.testing.TestImageLoaderModule
+import org.oppia.android.testing.TestLogReportingModule
+import org.oppia.android.testing.junit.InitializeDefaultLocaleRule
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.accessibility.AccessibilityTestModule
+import org.oppia.android.util.caching.AssetModule
+import org.oppia.android.util.caching.testing.CachingTestModule
+import org.oppia.android.util.gcsresource.GcsResourceModule
+import org.oppia.android.util.locale.LocaleProdModule
+import org.oppia.android.util.logging.EventLoggingConfigurationModule
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.SyncStatusModule
+import org.oppia.android.util.logging.firebase.FirebaseLogUploaderModule
+import org.oppia.android.util.networking.NetworkConnectionDebugUtilModule
+import org.oppia.android.util.networking.NetworkConnectionUtilDebugModule
+import org.oppia.android.util.parser.html.HtmlParserEntityTypeModule
+import org.oppia.android.util.parser.image.ImageParsingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Singleton
+
+/** Test for [AppCompatCheckBoxBindingAdapters] */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = AppCompatCheckBoxBindingAdaptersTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+
+class AppCompatCheckBoxBindingAdaptersTest {
+
+  @get:Rule
+  val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
+
+  @get:Rule
+  var activityRule: ActivityScenarioRule<AppCompatCheckBoxBindingAdaptersTestActivity> =
+    ActivityScenarioRule(
+      Intent(
+        ApplicationProvider.getApplicationContext(),
+        AppCompatCheckBoxBindingAdaptersTestActivity::class.java
+      )
+    )
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+    Intents.init()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private var colorRgb: Int = Color.valueOf(-0x10000).toArgb()
```

Comment:
Nit: please don't mix methods & variables (for consistency with other classes). Should be moved above ``setUp``.

Also, a negative color doesn't really make sense. Why can't this be a positive value?

Reply:
Ben, I have written this here because in the DrawableBindingAdaptersTest this line below the setUp and tearDown https://github.com/oppia/oppia-android/blob/c7349c3c24f99b16c9f5115fd410fbe9c827e1d0/app/src/sharedTest/java/org/oppia/android/app/databinding/DrawableBindingAdaptersTest.kt#L126

If it should be above the setUp then I can do that as well, Thanks.

Reply:
Ah, that's not right in the existing test, either. Thanks for fixing it here!

Reply:
Thanks.